ckan/controllers/admin.py

```diff
@@ -50,7 +50,7 @@ def _get_config_form_items(self):
             {'name': 'ckan.site_title', 'control': 'input', 'label': _('Site Title'), 'placeholder': ''},
             {'name': 'ckan.main_css', 'control': 'select', 'options': styles, 'label': _('Style'), 'placeholder': ''},
             {'name': 'ckan.site_description', 'control': 'input', 'label': _('Site Tag Line'), 'placeholder': ''},
-            {'name': 'ckan.site_logo', 'control': 'input', 'label': _('Site Tag Logo'), 'placeholder': ''},
+            {'name': 'ckan.site_logo', 'control': 'image_upload', 'label': _('Site Tag Logo'), 'placeholder': ''},
```

Comment:
@tino097 The form for the config options is created with `autoform.generate()` based on this dict, so you can pass the `is_upload_enabled` param here (with the value resulting from calling `h.upload_enabled()`, which you can call on this controller). Then modify `autoform.html` to use this value instead of the hardcoded value. Does this make sense?